nginx.conf

```diff
@@ -39,7 +39,15 @@ http {
   map_hash_bucket_size 256;
   map $host $sts {
     default "max-age=31536000; preload";
+
+    # enforce for subdomains, as a requirement for HSTS preload
+    # https://hstspreload.org/
     {{env "CLOUD_GOV_HOST"}} "max-age=31536000; includeSubDomains; preload";
+    code.gov "max-age=31536000; includeSubDomains; preload";
+    connect.gov "max-age=31536000; includeSubDomains; preload";
+    presidentialinnovationfellows.gov "max-age=31536000; includeSubDomains; preload";
+    sbst.gov "max-age=31536000; includeSubDomains; preload";
```

Comment:
This is the only domain that isn't owned by TTS, but [it doesn't seem like they have any subdomains besides `www`](https://github.com/18F/dns/blob/master/terraform/sbst.gov.tf).